README.md

```diff
@@ -79,6 +79,15 @@ where `LLVM_INSTALLATION_DIR` is
 the top-level directory
 where llvm is installed.
 
+### LLVM dependency for lit Regression tests
+
+ There are lit tests for the flang driver instance, these tests rely on the
+presence of llvm-lit and FileCheck, therefore these LLVM variables needs to
+be defined at cmake command line:
+
+ LLVM_EXTERNAL_LIT=<LLVM_INSTALLATION_DIR>/bin/llvm-lit
+ LLVM_TOOLS_BINARY_DIR= <LLVM_INSTALLATION_DIR>/bin
```

Comment:
nit: stray space here

Reply:
Can the cmake file be changed to use LLVM_INSTALLATION_DIR as a cmake variable and then infer the values for LLVM_DIR, LLVM_EXTERNAL_LIT and LLVM_TOOLS_BINARY_DIR from the value of LLVM_INSTALLATION_DIR?

Reply:
It looks like this:
cmake -DLLVM_DIR=$LLVM \$
      -DCMAKE_BUILD_TYPE=Debug \$
      -DLLVM_INSTALLATION_DIR=$LLVM_DIR \$
      -DCMAKE_INSTALL_PREFIX=$LLVM_DIR \$
      ../f18
Cannot find llvm-lit:
/bin/sh: 1: /llvm-lit: not found

Reply:
I believe that LLVM_EXTERNAL_LIT needs to belong in the cmake command. I've tried in the CMakeLists.txt and it did not worked very well. llvm-lit file was not being found properly